src/generators/worker.js

```diff
@@ -0,0 +1,10 @@
+import dispose from "./dispose";
+
+export default function worker(source) {
+  const url = URL.createObjectURL(new Blob([source], {type: "text/javascript"}));
+  const worker = new Worker(url);
```

Comment:
the API for [SharedWorker](https://github.com/conventional-changelog/standard-version) is super similar - maybe we could support either by making this `worker(source, constructor)`?